services/QuillLMS/db/migrate/20231206143410_create_pdf_subscriptions.rb

```diff
@@ -3,9 +3,8 @@
 class CreatePdfSubscriptions < ActiveRecord::Migration[7.0]
   def change
     create_table :pdf_subscriptions do |t|
-      t.string :title, index: true, null: false
       t.string :frequency, index: true, null: false
-      t.jsonb :filter_selections
+      t.references :admin_report_filter_selection, index: true, foreign_key: true, null: false
```

Comment:
We've been shying away from foreign keys due to concerns about migrations and performance, but that's mostly been around larger tables like User. I imagine that this one will not pie significant risks since it will only ever service admins.

Reply:
Foreign keys have been removed.